src/main/resources/GoogleApps/JSExtension.xml

```diff
@@ -111,50 +122,212 @@
         <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
       </use>
     </class>
+    <property>
+      <cache>short</cache>
+    </property>
+    <property>
+      <code>define('googleapps-buttons', {
```

Comment:
I'd add a suffix to the module name to make it more clear that it provides translation messages, e.g. ``googleapps-buttons-keys`` or ``googleapps-buttons-messages``